### ISSUE TITLE
Smart automatic modifiers

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#139]
Thanks for the option for modifiers! This can be slightly improved by automatically detecting static declarations and making them private:

Example:

```
static NSString *const XUCustomFieldNameKey = @"XUCustomFieldName";
```

Since the declaration states "static", it implies that it cannot be used outside of the scope of the document -> it can be declared private.

Current output:

```
let XUCustomFieldNameKey = "XUCustomFieldName"
```

Should be:

```
private let XUCustomFieldNameKey = "XUCustomFieldName"
```

**Comment by drkameleon** [comment]
Hmm... This sounds like a nice idea - I'll look into it! ;-)

----

P.S. I'm currently working (mostly) on app-related improvements (rather than enhancements in the converter itself) and coming with an... interesting update really soon. :) Re: the code conversion, feel free to post anything that I may have missed! (e.g. I noticed that I *might* have introduced a bug, in the last version, with the import statements and type recognition but still not sure...)

**Comment by charlieMonroe** [comment]
Just came across one more thing :)

**Comment by drkameleon** [comment]
Feature implemented as of the upcoming 2.3.
